server/helpers.js

```diff
@@ -16,11 +16,11 @@ module.exports.writeDatabase = (data) => {
 }
 module.exports.readDatabase = () => {
     return new Promise((resolve,reject) => {
-        fs.readFile('./database.json', "utf8", function (error,data) {
```

Comment:
This path also needs to be changed to the new one.

Reply:
Sorry, I'm confused, that is what I changed - I changed the path of database.json that appears twice in this file to the new directory. Are there any other files / places besides those two that it appears in?

Reply:
My bad. You are right!